README.md

```diff
@@ -94,6 +96,12 @@ Login Succeeded
 $ docker-compose -f docker-compose.yaml -f docker-compose-web-modeler-beta.yaml up -d
 ```
 
+To tear down the whole environment run the following command
+
+```
+$ docker-compose -f docker-compose.yaml -f docker-compose-web-modeler-beta.yaml down
```

Comment:
This will keep the volumes available, i.e. if I start up the environment again the data will be persisted, is this intentionally?

Reply:
No it's not. I wasn't aware of that. What would be the command to clean up the volumes as well?

Reply:
Add `-v` at the end. It may be worth to document both because sometimes users may want to keep the state of running workflows.